### PR TITLE
style(DEV-10393): align pagination arrows center

### PR DESCRIPTION
**Pull request by saladNights**
fixed the pagination arrow's alignment
Screenshot 2024-05-20 at 13 14 43

### CODE REVIEW

**Review by MadMed677 — APPROVED** [review]

File: .changeset/five-balloons-fail.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@monite/sdk-react': patch
+---
+
+align pagination arrows center
```
Comment (nekedos) [inline .changeset/five-balloons-fail.md:5]: Fix it, please, according to the rules:

### Changeset messages (changelog)

1. **Content:** Use present simple for entries.
2. **TASK-ID:** End each entry with the task identifier (e.g., *(DEV-1234)*).
3. **Examples:**
    - `fix(DEV-3456): add social network authentication feature`

File: packages/sdk-react/src/ui/table/TablePagination.tsx [hunk 2]
```diff
@@ -148,39 +141,41 @@ export const TablePagination = <T,>({
...
-          <StyledSelect
-            {...slotProps?.pageSizeSelect}
-            aria-label={t(i18n)`Rows per page`}
-            MenuProps={{
-              ...slotProps?.pageSizeSelect?.MenuProps,
-              container: root,
+        {hasPageSizeSelect && (
+          <Box
+            sx={{
+              position: 'absolute',
```
Comment (radist2s) [inline packages/sdk-react/src/ui/table/TablePagination.tsx:147]: `position: absolute` could work, but `<StyledSelect />` could be configured using the theme. Therefore it is important not to break the layout.

Here is the overrided `<StyledSelect />` example:
image

I would suggest only adding an extra blank cell to be used for centering.
Probably, usage of `<FormControl size="small"/>` also not needed

**Review by radist2s — CHANGES_REQUESTED** [review]
Thanks more for keeping things under control!

File: .changeset/five-balloons-fail.md (hunk 1 above)
Comment (saladNights) [reply .changeset/five-balloons-fail.md]: fixed

File: packages/sdk-react/src/ui/table/TablePagination.tsx (hunk 2 above)
Comment (saladNights) [reply packages/sdk-react/src/ui/table/TablePagination.tsx]: fixed

**Review by radist2s — APPROVED** [review]

**Pull request merged**